taskcluster/ci/browsertime/kind.yml

```diff
@@ -13,7 +13,7 @@ kind-dependencies:
 primary-dependency: signing
 
 only-for-build-types:
-    - nightly
+    - production
```

Comment:
does this include both fenix production and fennec production?

Reply:
`production` is only fenix production, which means "Firefox Nightly`. We only want to test nightly here. I thought it was a good idea to quickly transition these jobs to `org.mozilla.fenix` but it's causing more issues than I thought. I'm going to make this PR much simpler where we just deactivate the publication of `org.mozilla.fennec_aurora` and `org.mozilla.fenix.nightly` to GP while keep producing them.

We do need them produced for now, because some jobs in mozilla-central are actually expecting jobs marked as `nightly`. This is a bigger effort than I had anticipated. I'll take care of it once I'm back from PTO on July 13th.